app/models/notification_type.rb

```diff
@@ -9,6 +9,7 @@ class NotificationType < ApplicationRecord
   has_many :notifications
   validates :message, :presence => true
   validates :level, :inclusion => { :in => %w(success error warning info) }
+  validates :link_to, :inclusion => { :in => %w(subject initiator cause) }
```

Comment:
Should we add a value to `:link_to` for each type to the notification_type fixture? Also does this need to be set in a migration for existing types?

Reply:
@carbonin only if you need links for those notifications, it's perfectly fine if the `link_to` is not set, we were never using links in notifications before.

Reply:
Okay, I think this needs `:allow_blank => true` then.

Reply:
Specs should be fixed after adding the `allow_blank` from the previous comment.